src/endpoint/types.ts

```diff
@@ -43,7 +43,7 @@ export interface Endpoint {
   DEFAULTS: EndpointDefaults
   defaults(endpointOptions: EndpointParams): Endpoint
   merge(
-    endpointRoute: string,
+    endpointRoute: string | EndpointOptions,
```

Comment:
This doesn't seem right. Why was it changed? 🤔

Reply:
Yeah, unnecessary change, thanks.